sdk/test-utils/test-utils/src/tracing/testTracingSpan.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  TracingSpan,
+  SpanStatus,
+  TracingSpanOptions,
+  TracingSpanKind,
+  TracingContext,
+  TracingSpanContext
+} from "@azure/core-tracing";
+
+export class TestTracingSpan implements TracingSpan {
+  name: string;
+  spanKind: TracingSpanKind | undefined;
+  /** Parent tracing context or existing */
```

Comment:
We should probably add doc comments to the public members here.

Reply:
done